docs/schema.graphql

```diff
@@ -0,0 +1,158 @@
+## NOTE: This schema was generated by Graphback and should not be changed manually
```

Comment:
Should remove Graphback references in main doc to remain impartial.

Reply:
Actually I should remove this file :D